### PR TITLE
Correcting comments in T5Stack to reflect correct tuple order 

### PR DESCRIPTION
**Pull request by talkhaldi**
# What does this PR do?

In order to match the actual order (line 513 and 516, and as accessed in 968), I've changed the order mentioned in comments L962 and L966-967.

## Before submitting
- [ x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR. (Seems like @patrickvonplaten, @patil-suraj are suggested for T5)

### CODE REVIEW

**Review by patil-suraj — APPROVED** [review]
Thanks for doing this!

Could you also change the comment here?
https://github.com/huggingface/transformers/blob/95dab34d5588fb155dfed8293ac2fbb1217a95a7/src/transformers/models/t5/modeling_t5.py#L693

**Comment by patil-suraj** [comment]
Hi, @talkhaldi  thanks for correcting the other comment!

There is one code quality check failing, could run `make style && make quality` and push the code again?

**Review by patrickvonplaten — APPROVED** [review]

**Comment by talkhaldi** [comment]
Hi!

The command you mentioned was requiring a lot of dependencies, some of which conflicted with the venv I was using, so instead of creating a new venv to run it, I removed the extra trailing space which technically was the only extra thing, and hoped it would work, but seems not ^^' 

I created a new venv and running the command now...

**Comment by talkhaldi** [comment]
Hi @patil-suraj,

So it seems make style works well, but I get this error with make quality:

```
> make quality
black --check� �amples tests src utils
839 files would be left unchanged.
isort --check-only examples tests src utils
python utils/custom_init_isort.py --check_only
flake8 examples tests src utils
make extra_quality_checks
make[1]: Entering directory '/mnt/berry/home/alkhaldi/contribcode/transformers'
python utils/check_copies.py
python utils/check_table.py
python utils/check_dummies.py
python utils/check_repo.py
Traceback (most recent call last):
  File "utils/check_repo.py", line 22, in <module>
    from transformers.models.auto import get_values
ImportError: cannot import name 'get_values' from 'transformers.models.auto' (unknown location)
make[1]: *** [Makefile:33: extra_quality_checks] Error 1
make[1]: Leaving directory '/mnt/berry/home/alkhaldi/contribcode/transformers'
make: *** [Makefile:42: quality] Error 2
```

Do you have an idea of what's wrong?

**Comment by patil-suraj** [comment]
hi @talkhaldi

please make sure you have all the dev deps before running make

to do that you could run `pip install -e ".[dev]"` from the root of the repo.

**Comment by talkhaldi** [comment]
Hi @patil-suraj,

I have applied that command successfully, but it seems the tests are still failing :o What could be the problem?

Note: That command changed many files, but I only added modeling_t5.py to the commit. Should I include the changes of all files even though I haven't changed them manually?

**Comment by patil-suraj** [comment]
Hi @talkhaldi 

I think this is because we upgraded the version of `black`. Could you rebase your branch with master and then push again ?

**Comment by talkhaldi** [comment]
Hi @patil-suraj,

The check_code_quality test passes now, but another test fails. Can you please tell me what's wrong?

File: src/transformers/models/t5/modeling_t5.py [hunk 1]
```diff
@@ -700,7 +700,8 @@ def forward(
         else:
             outputs = outputs + attention_outputs
 
-        return outputs  # hidden-states, present_key_value_states, (self-attention weights), (self-attention position bias), (cross-attention weights), (cross-attention position bias)
+        outputs = outputs + (present_key_value_state,) + attention_outputs
```
Comment (patrickvonplaten) [inline src/transformers/models/t5/modeling_t5.py:703]:
```suggestion
```

This was not intentional no?

Comment (patrickvonplaten) [reply src/transformers/models/t5/modeling_t5.py:703]: The rest looks good to me

Comment (talkhaldi) [reply src/transformers/models/t5/modeling_t5.py]: @patrickvonplaten Yes, that was an accident. Thank you!

**Pull request merged**